spec/controllers/idv/address_controller_spec.rb

```diff
@@ -63,24 +63,6 @@
       end.to change { subject.idv_session.address_edited }.from(nil).to eql(true)
     end
 
-    it 'updates pii_from_doc in idv_session' do
```

Comment:
Maybe we should invert this test (assert that pii_from_doc is not modified?)

Reply:
I am a little resistant because I am hopeful we can make `pii_from_doc` immutable here. That should reduce the risk of a regression and render this test unnecessary.

Reply:
Works for me!